components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Cdm/src/Microsoft.Azure.IIoT.OpcUa.Cdm.csproj

```diff
@@ -3,12 +3,10 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Azure Industrial IoT Core Azure Cdm Implementations</Description>
-    <VersionSuffix>$(PrereleaseSuffix)</VersionSuffix>
     <NoWarn>NU1608;NU5125;NU1701</NoWarn>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.CommonDataModel.ObjectModel" Version="1.0.1" />
-    <PackageReference Include="Microsoft.CommonDataModel.ObjectModel.Adapter.Adls" Version="1.0.1" />
```

Comment:
don't we still need the CDM's sdk Adls adapter?

Reply:
There is a single adapter now that uses the datalake filesystem sdk and drive/file/folder abstractions.   This makes it possible to shoehorn in other storage e.g. local file system or database without changing the cdm processor.   Also good for testing.